src/main/java/org/terasology/basicRendering/rendering/AdvancedRenderingModule.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.basicRendering.rendering;
```

Comment:
Should we go for `org.terasology.basicrendering.rendering` instead? Has bugged me on other modules, too.

See https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

```suggestion
package org.terasology.basicrendering.rendering;
```